packages/react-components/react-carousel-preview/library/src/components/CarouselSlider/CarouselSlider.types.ts

```diff
@@ -0,0 +1,47 @@
+import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
+import { MutableRefObject } from 'react';
+
+export type CarouselSliderSlots = {
+  /**
+   * The div that will be animated to track and center current index
+   */
+  root: Slot<'div'>;
+  /**
+   * The viewport/window of the carousel
+   */
+  container: NonNullable<Slot<'div'>>;
+};
+
+/**
+ * CarouselSlider Props
+ */
+export type CarouselSliderProps = Partial<ComponentProps<CarouselSliderSlots>> & {
+  /**
+   * Used to define the carousel card size, can be any css metric i.e. 100% / 100px / 100vw
```

Comment:
This comment and variable name are a little unclear to me? Does it control the peak amount?

Reply:
This will control how large each carousel card is in proportion to the viewport, so if you want each card to take up the full width it would be '100%', if you want the other cards to peek in ~10% then the value would be `80%` to leave space on each side - users could also enforce a pixel width of cards if desired.

Reply:
Coolio - it might be worth adding to the comment to help others understand.